test/jdk/java/awt/a11y/AccessibleActionsTest.java

```diff
@@ -92,6 +94,31 @@ void createTree() {
     super.createUI(panel, "AccessibleActionsTest");
   }
 
+  private void createEditableTextArea() {
+    AccessibleComponentTest.INSTRUCTIONS = "INSTRUCTIONS:\n"
+            + "Check a11y show context menu in editable JTextArea.\n\n"
+            + "Turn screen reader on and press Tab to move to the text area and vo+shif+m.\n\n"
```

Comment:
Just a small nit, can we use the same style of instructions, the previous tests use a separate text to press the VO shortcuts, here it could be something like this:
"Turn screen reader on and press Tab to move to the text area"
"Perform the VO action \"Open a shortcut menu\" (VO+Shift+m)\n\n"

Reply:
done